modules/axp/Animal/lib/RaceHorse.pm

```diff
@@ -215,4 +108,6 @@ EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 =cut
 
+__PACKAGE__->meta->make_immutable( inline_constructor => 0, inline_destructor => 0 );
```

Comment:
Yeah, you need an `inline_destructor` now as well to avoid warnings, since we defined `DESTROY`.